smarts/env/wrappers/format_obs.py

```diff
@@ -318,7 +315,7 @@ def get_spaces() -> Dict[str, Callable[[Any], gym.Space]]:
     """Returns the available gym spaces of a `StdObs`.
 
     Returns:
-        Dict[str, Callable[[Any], gym.Space]]:
+        Dict[str, Callable[[Any], S]]:
```

Comment:
What is `S`?

Reply:
It was a template type that I had reverted.